lib/iris/fileformats/um/_fast_load.py

```diff
@@ -373,16 +373,18 @@ def structured_um_loading():
         fields of each phenomenon, such that different phenomena may have
         different field structures, and can be interleaved in any way at all.
 
+        .. note::
+
+             At present, fields with different values of 'LBUSER5'
+             (pseudo-level) are *also* treated as different phenomena.
+             This is a temporary fix, standing in place for a more correct
```

Comment:
i don't think i would use the term 'temproary fix'

I would prefer to state what the behaviour is and how it will be experienced.

perhaps
`This will result in separate cubes for differnet values of LBUSER, with a coordinate representing the value.  These can be concatenated/merged to form cubes with a pseudo level dimension.`
or somesuch